Lib/test/test_threaded_import.py

```diff
@@ -2,8 +2,8 @@
 # Demo/threads/bug.py.  It simply provokes a number of threads into
 # trying to import the same module "at the same time".
 # There are no pleasant failure modes -- most likely is that Python
-# complains several times about module random having no attribute
-# randrange, and then Python hangs.
+# complains several times about module types having no attribute
+# ModuleType, and then Python hangs.
```

Comment:
I can only say that the comment was written 16 years ago by @tim-one, commit aa222234c0836d32030c667240f20d8879e7f6b3... The import machinery was deeply modified since this commit was made... I'm not sure that the test still makes sense :-p